mm2src/adex_cli/src/log.rs

```diff
@@ -0,0 +1,18 @@
+use log::LevelFilter;
+use log4rs::{append::console::ConsoleAppender,
+             config::{Appender, Root},
+             encode::pattern::PatternEncoder,
+             Config};
+
+const REDUCED_LOG_FORMAT: &str = "{l}: {m}{n}";
+
+pub fn init_logging() {
```

Comment:
It's not strictly required for `adex` tool because all the messages should be shown regardless if those are errors or info all of them are supposed to be shown. And we can use env_logger of coarse